src/master.py

```diff
@@ -141,12 +103,109 @@ def work():
     bootstrap.update_ec2_stack(stackname, context, concurrency='serial')
     return True
 
-# TODO: extract just the salt update part from `remaster`
 @debugtask
 @requires_aws_stack
 def update_salt(stackname):
-    current_master_stack = core.find_master_for_stack(stackname)
-    return remaster(stackname, current_master_stack)
+    "updates the installed version of salt to the version specified in the project file. see `docs/salt.md`"
+    # TODO: turn this into a decorator
+    import cfn
```

Comment:
`cfn` is a module with `@task`s, it shouldn't be imported from other modules because you get duplicated tasks:
```
    masterless.launch
    masterless.set_versions               updates the cloned formulas on a masterl...
    masterless.cfn.aws_stack_list         returns a list of realized stacks. does ...
    masterless.cfn.cmd
    masterless.cfn.destroy                Delete a stack of resources.
    ...
```
`_check_want_to_be_running` should be moved to `buildercore.lifecycle` but there is a `utils._pick` calls to address. I'll make a separate PR to simplify it so that you can just `import buildercore.lifecycle` here.